### ISSUE TITLE
Error when importing Thermo raw files

### ISSUE THREAD

**Issue by CapriniEvaristo** [mzmine/mzmine2#697]
### I can not import Thermo raw files

* My operating system: windows 10
* My MZmine version: 2.51

### When I select the data appears an error message: identical another colleague has reported for Waters raw files this week.

![image](https://user-images.githubusercontent.com/54907762/64286574-ca888a00-cf2b-11e9-9e6e-480696078df9.png)
![image](https://user-images.githubusercontent.com/54907762/64286738-189d8d80-cf2c-11e9-884c-efa83fb9f1c4.png)

[15:49:55|SEVERE|net.sf.mzmine.taskcontrol.impl.WorkerThread]: Error of task Opening file C:\Users\xxxxxxxxxxxxxxxxxxxxxxxxxx.raw: java.io.IOException: System.Runtime.InteropServices.COMException (0x800736B1): Falha na recupera??o de factory de classes COM do componente com CLSID {1D23188D-53FE-4C25-B032-DC70ACDBDC02}  (NativeFileReadTask.java:198)6b1.

**Comment by tomas-pluskal** [comment]
Yes, some people have reported a similar error in the past. To be honest, I don't know why it happens, it seems to happen only on some Windows systems but not on others. Try installing MSConvert from the Proteowizard package, this error might disappear after you install MSConvert. Or maybe not. In any case, you can use MSConvert to convert your data to mzML.

**Comment by tomas-pluskal** [comment]
It seems there is a new tool now that allows reading Thermo Raw files also on Linux and Mac via Mono: https://github.com/compomics/ThermoRawFileParser

I will try to switch MZmine to using this tool in near future.

**Comment by Kunzol** [comment]
**Visual Studio Runtime 2008 32 bit** is needed to run the thermo raw file reader, which is in the lib directory.
